web/api/health/README.md

```diff
@@ -45,6 +45,7 @@ The following will return an SVG badge of the alarm named `NAME`, attached to th
 ## Health Management API
```

Comment:
Did my first test and it failed:

```
[christopher@chris-msi netdata]$ curl -s "http://localhost:19999/api/v1/manage/health?cmd=LIST" -H "X-Auth-Token: cc3598fc-578a-11e9-8810-b808cfc55d7d" 2>&1
{
	"all": true,
	"type": "DISABLE",
	"silencers": []
}
                       
[christopher@chris-msi netdata]$ cat /var/lib/netdata/health.silencers.json 
{
	"all": true,
	"type": "DISABLE",
	"silencers": []
}
```

This far it's good, so it picked up the config from the file I already had there, when netdata started.

```
[christopher@chris-msi netdata]$ curl -s "http://localhost:19999/api/v1/manage/health?cmd=RESET" -H "X-Auth-Token: cc3598fc-578a-11e9-8810-b808cfc55d7d" 2>&1
All health checks and notifications are enabled
[christopher@chris-msi netdata]$ cat /var/lib/netdata/health.silencers.json 
{
	"all": true,
	"type": "DISABLE",
	"silencers": []
}
[christopher@chris-msi netdata]$ curl -s "http://localhost:19999/api/v1/manage/health?cmd=LIST" -H "X-Auth-Token: cc3598fc-578a-11e9-8810-b808cfc55d7d" 2>&1
{
	"all": false,
	"type": "None",
	"silencers": []
}
```

Not good. The change was not written to the file.

Reply:
I changed the script and now I am doing checks more than one time, I had success in all test that it was done, please verify that everything is all right now!